pkg/edgeview/src/basics.go

```diff
@@ -0,0 +1,451 @@
+			helpExample("trace", "traceroute to www.google.com and to zedcloud server", true)
+			helpExample("trace/www.microsoft.com", "run traceroute to www.microsoft.com", false)
+		case "url":
+			helpOn("url", "display url metrics for zedclient, zedagent, downloader and loguploader")
+		case "wireless":
+			helpOn("wireless", "display the iwconfig wlan0 info and wpa_supplicant.conf content")
+		// system
+		case "configitem":
+			helpOn("configitem", "display the device configitem settings, highlight the non-default values")
+		case "cp":
+			helpOn("cp/<path>", "copy file from the device to locally mounted directory by specify the path")
+			helpExample("cp//config/device.cert.pem", "copy the /config/device.cert.pem file to local directory", true)
+			helpExample("cp//persist/newlog/keepSentQueue/dev.log.1630451424116.gz", "copy file with path to local directory", false)
+		case "datastore":
+			helpOn("datastore", "display the device current datastore: EQDN, type, cipher information")
+		case "download":
+			helpOn("download", "display the download config and status during downloading operation and url stats since reboot")
+		case "hw":
+			helpOn("hw", "display the hardware from lshw information in json format")
+		case "model":
+			helpOn("model", "display the hardware model information in json format")
+		case "newlog":
+			helpOn("newlog", "display the newlog statistics and file information in each of the newlog directory and disk usage")
+		case "pci":
+			helpOn("pci", "display the lspci information on device")
+		case "ps":
+			helpOn("ps/<string>", "display the process status information on matching string")
+			helpExample("ps/containerd", "display the processes with name of containerd", true)
+		case "cipher":
+			helpOn("cipher", "display cipher information on datastore, device and controller certificates, etc.")
+		case "usb":
+			helpOn("usb", "display the lsusb information on device")
+		case "shell":
+			helpOn("shell/<some command>", "run shell on the command supplied on the device")
+			helpExample("'shell/ls -l /run/nim'", "run 'ls -l /run/nim' command on device", true)
+			helpExample("'shell/cat /config/server'", "run 'cat /config/server' command on device", false)
+		case "volume":
+			helpOn("volume", "display the app volume and content tree information for each app")
+		// log
+		case "log":
+			helpOn("log/<search string> [-time <start>-<end>] [-json] [-type <app|dev>]", "display log with search-string, default is now to 30 mins ago")
+			helpExample("log/panic -time 0.2-2.5", "display log contains 'panic', from 0.2 to 0.5 hours ago", true)
+			helpExample("log/Clock -type app", "display previous 30 minutes log contains 'Clock' in app log", false)
+			helpExample("log/certificate -time 2021-08-15T23:15:29Z-2021-08-15T22:45:00Z -json",
+				"display log during the specified time in RFC3339 format which contains 'certificate' in json format", false)
+		default:
```

Comment:
I think a command showing reboot-reason and reboot-stack would be also helpful.

Reply:
i could not find where to look for this on the device, @milan-zededa do you know where it is saved?

Reply:
They get stored under `/persist/reboot-reason` and `/persist/reboot-stack`.